antd.customize.less

```diff
@@ -154,7 +154,9 @@
   0,
   0,
   98%
-); // background of header and selected item
+); 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5489110)